src/Components/creatorPage/AuthorInfoCard.js

```diff
@@ -0,0 +1,158 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import Box from '@mui/material/Box';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import '../../styles/fonts.css';
+
+const styles = {
```

Comment:
Your sx solution is so beautiful, thank you for your hard work!! I might ask the other devs to use your code as an example if we have time to fully standardize the code, it's so clean!